### ISSUE TITLE
RenderManager(0): WorkerThread(0): Exception while doing some work!

### ISSUE THREAD

**Issue by ghost** [BlueMap-Minecraft/BlueMap#569]
### What i did / Steps to reproduce

Installed Bluemap 5.2 on my server and left it running rendering for a few weeks.

Agreed with ELUA.
Web server is running.

### Expected result

it kept saying forever
```
[Tue 12:01:29 ERROR BlueMap] RenderManager(0): WorkerThread(0): Exception while doing some work!
java.lang.IllegalArgumentException: newLimit < 0: (-163840 < 0)
    at java.base/java.nio.Buffer.createLimitException(Buffer.java:406)
    at java.base/java.nio.Buffer.limit(Buffer.java:380)
    at java.base/java.nio.ByteBuffer.limit(ByteBuffer.java:1565)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.core.world.mca.region.MCARegion.readFully(MCARegion.java:190)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.core.world.mca.region.MCARegion.readFully(MCARegion.java:183)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.core.world.mca.region.MCARegion.iterateAllChunks(MCARegion.java:158)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.core.world.mca.MCAWorld.preloadRegionChunks(MCAWorld.java:194)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.core.world.World.preloadRegionChunks(World.java:92)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.common.rendermanager.WorldRegionRenderTask.init(WorldRegionRenderTask.java:142)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.common.rendermanager.WorldRegionRenderTask.doWork(WorldRegionRenderTask.java:162)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.common.rendermanager.CombinedRenderTask.doWork(CombinedRenderTask.java:56)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.common.rendermanager.RenderManager.doWork(RenderManager.java:318)
    at BlueMap-5.2-paper.jar//de.bluecolored.bluemap.common.rendermanager.RenderManager$WorkerThread.run(RenderManager.java:343)
```

### Actual result

I can enter the website and the graphics are rendered correctly.

However, it keeps spitting errors into the console forever.

### Context
**BlueMap Version:** 
`BlueMap-5.2-paper.jar`

**OS:** 
`Ubuntu 24.04 LTS`

**Server-Version**:  
`Paper version git-Paper-496 (MC: 1.20.4) (Implementing API version 1.20.4-R0.1-SNAPSHOT) (Git: 7ac24a1 on ver/1.20.4)`

**Comment by TBlueF** [comment]
Duplicate of #560 

This has already been fixed with 6032043126feb882a518c673346aa3251d137e7e, there is just no release with the fix yet.
You can use the [latest development build](https://github.com/BlueMap-Minecraft/BlueMap/actions/runs/9748943199) if you don't want to wait for the next release (`Artifacts` button to download) :)

**Comment by ghost** [comment]
Thank you for your reply.

Sorry, this post was a duplicate, I didn't check enough.

Thank you for your quick response.
Please take care of yourself and continue your good development!
